config/mail.php

```diff
@@ -60,15 +60,6 @@
 			'transport' => 'sendmail',
 			'path' => '/usr/sbin/sendmail -bs',
 		],
-
-		'log' => [
-			'transport' => 'log',
-			'channel' => env('MAIL_LOG_CHANNEL'),
-		],
-
-		'array' => [
-			'transport' => 'array',
-		],
```

Comment:
What happens, if these array keys are not set? Are you sure that they are really not used anywhere? This time I was to lazy to look into the vendor source code myself.